ibis/expr/datatypes.py

```diff
@@ -602,6 +603,38 @@ def __str__(self) -> str:
         )
 
 
+class GeoSpatial(DataType):
```

Comment:
Is there some shared functionality among all child classes of this class? If not, then we should just make the geospatial types subclass from `DataType` and not tie them together unnecessarily.

Reply:
I thought to create the GeoSpatial data type just to generalize the functions parameters, for example: 

```
ST_Distance(poly1, ST_GeomFromText('POINT(0 0)'))
```

where st_distance returns shortest planar distance between geometries

maybe would be better to change GeoSpatial name to Geometry

what do you think?

Reply:
actually I will think more about this .. postgis use geometry and geography types ...